portfolio/src/main/webapp/script.js

```diff
@@ -60,7 +60,6 @@ function toggleBlogPost(buttonId) {
  */
 function fetchComments() {
   fetch('/data').then(response => response.json()).then((commentData) => {
-    console.log(commentData)
```

Comment:
This seems unrelated?

Reply:
Yeah, it doesn't affect the main functionality I implemented in the commit. But I didn't know whether a whole new commit to remove the console.log was too excessive, so I kept it in this commit - should I have done it separately?

Reply:
I realize it seems minor, but it's pushing it quite a bit to not split this out into a new change. I think it would be borderline acceptable to remove it in this commit if code adjacent the line being deleted was being modified in this change as well, but in this case it's a whole new file.

Reply:
Yep, that makes sense, I'll keep that in mind in the future.